cmd/snapshotgc/snapshotgc.go

```diff
@@ -182,3 +249,31 @@ func deleteSnapshots(
 		}
 	}
 }
+
+func main() {
+	var prSnapshotsToKeep, nonPrSnapshotsToKeep int
+	flag.IntVar(
+		&prSnapshotsToKeep,
+		"pr-snapshots-to-keep",
+		5,
```

Comment:
Please us safer defaults, someting like 512 so we don't wipe out everything accidentally when misconfigured

Reply:
done